Python Examples/TestMonitor/DeleteResults/delete_results.py

```diff
@@ -0,0 +1,81 @@
+import uuid
+import sys
+import os
+import datetime
+
+current = os.path.dirname(os.path.realpath(__file__))
+parent = os.path.dirname(current)
+sys.path.append(parent)
+
+import test_data_manager_client
+
+def get_test_result():
+    test_result = {
+        "programName": "Power Test",
+        "status": {
+            "statusType": "RUNNING",
```

Comment:
I think creating results with `status` as `PASSED` and `totalTimeInSeconds` as some value will be more realistic.

Reply:
Updated the status as passed and the total time to 13.6 seconds